lib/cdo/honeybadger.rb

```diff
@@ -84,4 +84,29 @@ def self.parse_exception_dump(error)
 
     [error_message, error_lines]
   end
+
+  # Returns all issues (across cronjobs, dashboard, and pegasus) that have occured since midnight.
+  # @return [Array[Hash]] An array of hashes summarizing the recent issues.
+  def self.get_recent_issues
+    issues = []
+    midnight_epoch = Time.now.to_i / 86400 * 86400
+
+    {cronjobs: 45435, dashboard: 3240, pegasus: 34365}.each do |project, project_id|
+      api_url = "https://app.honeybadger.io/v2/projects/#{project_id}/faults"\
+        "?occurred_afer=#{midnight_epoch}&q=-is:resolved%20-is:paused%20-is:ignored"
+      response = `curl -u #{CDO.honeybadger_api_token}: #{api_url}`
+      parsed_response = JSON.parse response
+      parsed_response['results'].each do |issue|
+        issues << {
+          environment: issue['environment'] ? issue['environment'] : 'unknown',
```

Comment:
For some reason, the environment is always set in the API response. May or may not be worth exploring.